### ISSUE TITLE
Broken CSS Build in pdfjs-dist (v2.15.349)

### ISSUE THREAD

**Issue by hkleungai** [mozilla/pdf.js#15260]
### Configuration
- PDF.js version: v2.15.349

### What went wrong? (add screenshot)

There seems to be a new image import `images/toolbarButton-editorInk.svg` in
- [`pdfjs-dist/legacy/web/pdf_viewer.css`](https://github.com/mozilla/pdfjs-dist/blob/80e9ead6a60fd3a53c2795fab864101962250dff/legacy/web/pdf_viewer.css#L714), and
- [`pdfjs-dist/web/pdf_viewer.css`](https://github.com/mozilla/pdfjs-dist/blob/80e9ead6a60fd3a53c2795fab864101962250dff/web/pdf_viewer.css#L714) 

I can find this svg file in the [pdfjs github page assets](https://github.com/mozilla/pdf.js/blob/gh-pages/web/images/toolbarButton-editorInk.svg), but neither [`pdfjs-dist/legacy/web/images`](https://github.com/mozilla/pdfjs-dist/blob/80e9ead6a60fd3a53c2795fab864101962250dff/legacy/web/images) nor [`pdfjs-dist/web/images`](https://github.com/mozilla/pdfjs-dist/blob/80e9ead6a60fd3a53c2795fab864101962250dff/web/images) are updated with the svg file in v2.15.349.

It is causing weird build issue on `pdfjs-dist`.

**Comment by donarus** [comment]
is there any estimation when a version with this fix will be released ? We need other fixes in 2.15.349 but for now we have to hack the build by manually copying the SVG vile into appropriate node module within node_modules

**Comment by Snuffleupagus** [comment]
> is there any estimation when a version with this fix will be released ? We need other fixes in 2.15.349 but for now we have to hack the build by manually copying the SVG vile into appropriate node module within node_modules

Sorry, but I don't consider this CSS-only issue serious enough to warrant creating a new release immediately (given the work involved in doing so) considering that the last release was less than a week ago. Hence this will, once landed, simply be included in the next upcoming release (in about a month).

**Comment by donarus** [comment]
Sorry, but I would disagree. I agree that it would not make sense to release the new version with all the other  (and potentially untested) features and fixes. But on the other side, this may be blocking issues for many other related projects and the fix could be created from the actual release branch just patching it by adding the mentioned file to the correct place and rerelease it for example under 2.15.400 or 2.15.349.2. But if this solution is not a way, tell me a reason why please.

Until fixed, we figured out how to get over it by using the simple hack for NPM based builds: 

**package.json**
```json
{
  ...
  "scripts": {
    .....
    "postinstall": "node postinstall-fixes.js"
  },
  ...
}
```

**postinstall-fixes.js**
```
const file = fs.createWriteStream("./node_modules/pdfjs-dist/legacy/web/images/toolbarButton-editorInk.svg");
const request = https.get("https://raw.githubusercontent.com/mozilla/pdf.js/edc9ad13bf8f422f9495c4ede9bb9989969326e4/web/images/toolbarButton-editorInk.svg", function(response) {
   response.pipe(file);
   file.on("finish", () => {
       file.close();
   });
});
```

But still, it would make sense to fix this issue because many others would need to solve it somehow.

**Comment by Tucaen** [comment]
Shouldn't the path be `./node_modules/pdfjs-dist/web/images/toolbarButton-editorInk.svg` without the "legacy" part?
With "legacy" it's not working for me.

**Comment by donarus** [comment]
we use the legacy build. the same should work for a regular one. Thanks for the pointing it out.

**Comment by phloose** [comment]
> Sorry, but I don't consider this CSS-only issue serious enough to warrant creating a new release immediately

Hmm this means, you consider a failing build for a lot of users not serious enough to release a new version that fixes this issue?! Sounds a bit harsh to me... What is the culprit here? What would take so much time to make sure the file exists or the path is correct? A patch for this would surely help a lot of users

**Comment by timvandermeij** [comment]
The fix for this is available in version 2.16.105 which we just released.
